app/scss/_base.scss

```diff
@@ -47,8 +48,14 @@ p {
     color: $teal;
     font-size: 2em;
     font-weight: 400;
+    line-height: 1.5;
+    margin: $xl $sm $md $sm;
     @include breakpoint(small-screen) {
       font-size: 1.5em;
+      margin: $lg $md $sm $sm;
+    }
+    @media screen and (min-width: 1000px) {
```

Comment:
Create another breakpoint mixin like the above so that it uses the same pattern. If you want to switch to all min-width, that's fine. I did max width since we started with a desktop design rather than starting with a mobile design.